auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue

```diff
@@ -0,0 +1,506 @@
+        // check if update didn't succeed
+        if (businessResponse?.status !== StatusCodes.OK) {
+          this.$emit('add-unknown-error')
+        }
+        // update folio number
+        await this.updateFolioNumber({
+          businessIdentifier: this.businessIdentifier,
+          folioNumber: this.folioNumber
+        })
+        // let parent know that add was successful
+        this.$emit('add-success')
+      } catch (exception) {
+        if (exception.response?.status === StatusCodes.UNAUTHORIZED) {
+          this.$emit('add-failed-invalid-code', this.passcodeLabel)
+        } else if (exception.response?.status === StatusCodes.NOT_FOUND) {
+          this.$emit('add-failed-no-entity')
+        } else if (exception.response?.status === StatusCodes.NOT_ACCEPTABLE) {
+          this.$emit('add-failed-passcode-claimed')
+        } else if (exception.response?.status === StatusCodes.BAD_REQUEST) {
+          console.log(exception.response?.data?.code === 'DATA_ALREADY_EXISTS')
+          this.$emit('business-already-added', { name: this.businessName, identifier: this.businessIdentifier })
+        } else {
+          this.$emit('add-unknown-error')
+        }
+      } finally {
+        this.resetForm()
+      }
+    }
+  }
+
+  resetForm (emitCancel = false): void {
+    this.passcode = ''
+    this.folioNumber = ''
+    this.authorizationName = ''
+    this.passcodeOption = false
+    this.emailOption = false
+    this.requestAuthBusinessOption = false
+    this.requestAuthRegistryOption = false
+    this.$refs.addBusinessForm.resetValidation()
+    this.isLoading = false
+    if (emitCancel) {
+      this.$emit('on-cancel')
+    }
+    this.dialog.show = false
+  }
+
+  formatBusinessIdentifier (): void {
+    this.businessIdentifierRules = [
+      v => !!v || 'Incorporation Number or Registration Number is required',
+      v => CommonUtils.validateIncorporationNumber(v) ||
+        'Incorporation Number or Registration Number is not valid'
+    ]
+    // this.businessIdentifier = CommonUtils.formatIncorporationNumber(this.businessIdentifier)
+  }
+
+  openHelp (): void {
+    this.$refs.helpDialog.open()
+  }
+
+  /** Emits event to parent initially and when business identifier changes. */
+  @Watch('businessIdentifier', { immediate: true })
+  private onBusinessIdentifierChange (): void {
+    this.$emit('on-business-identifier', this.businessIdentifier)
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '$assets/scss/theme.scss';
+
+::v-deep {
+
+  .v-list-group{
+    border-bottom: 1px solid rgb(228, 228, 228);
+    &.top-of-list{
+      border-top: 1px solid rgb(228, 228, 228);
+    }
+    .item-content{
+      color: #000 !important;
+    }
+  }
+
+  .v-list-item{
+    background: #FFFFFF;
+    height: 4rem !important;
+    margin: 0 !important;
+  }
+
+  .v-list-item--link>
+  .v-list-item__title{
+    font-weight: 300 !important;
+    margin-left:-1rem !important;
+    color: var(--v-primary-base) !important;
```

Comment:
This is a way to get the Vuetify colour, I think, but there is probably an existing colour variable you can use more simply here (something like `color: $app-blue`).